tests/crescent/internal/test_handle_resp.py

```diff
@@ -23,9 +23,9 @@ async def test_command(ctx: Context):
         interaction=CommandInteraction(
             app=bot,
             id=None,
-            application_id=bot.application_id,
+            application_id=...,
             type=InteractionType.APPLICATION_COMMAND,
-            token=bot.token,
```

Comment:
What's the reason for this change?

Reply:
I thought it made it more clear that `application_id` doesn't matter. I also removed `bot.application_id` because it shouldn't have been there imo.